tests/llmobs/test_experiments.py

```diff
@@ -0,0 +1,46 @@
+"""
+To run these tests, you need to set the following environment variables:
+
+- RECORD_REQUESTS=1  # used to delay tests until data is ready from the backend
+- DD_APP_KEY=...  # your datadog application key
+- DD_API_KEY=...  # your datadog api key
+
+and must have the test agent (>=1.27.0) running locally and configured to use the vcr cassette directory
+
+eg. VCR_CASSETTES_DIRECTORY=tests/cassettes ddapm-test-agent ...
+"""
+
+import os
+
+import pytest
+
+
+@pytest.fixture
+def test_dataset(llmobs):
+    ds = llmobs.create_dataset(name="test-dataset", description="A test dataset")
+
+    # When recording the requests, we need to wait for the dataset to be queryable.
```

Comment:
curious q but do we need to wait even for vcr?

Reply:
we don't need to wait when we have the vcr, just when we're recording them for the first time (or on regeneration)